packages/notebook/src/actions.tsx

```diff
@@ -1492,6 +1492,9 @@ export namespace NotebookActions {
             typeof cell.id === 'string'
               ? cell.id
               : undefined;
+          if (cell.metadata.hasOwnProperty('execution')) {
+            delete cell.metadata['execution'];
+          }
```

Comment:
1. This should definitely not run for cut-paste scenario as we would lose timing information
2. I am not sure if we want to run this even for copy-paste; when copy-pasting across notebook we may want to preserve timing info of cells; it is only warranted if cell has not completed execution
3. I think that this should be addressed in `jupyterlab-execute-time` instead: if execution counter is idle it should not show the ticking clock for any given cell.